apps/files/apps.py

```diff
@@ -0,0 +1,21 @@
+from django.apps import AppConfig
+from django.db.models.signals import pre_delete
+
+
+class FilesConfig(AppConfig):
+    name = "apps.files"
+    label = "files"
+
+    def ready(self):
+        from .models import File
+
+        for field in File._meta.get_fields():
+            if not field.many_to_many:
+                continue
+
+            def delete_orphaned_files(sender, m2m_field=field, **kwargs):
```

Comment:
There's currently no way to share files between assistants except using Django Admin or the a Django shell.

I used a many-to-many relation because I expect the files model to be used in other places as well and I didn't want to put a generic foreign key on `Files`.

I guess I could add a check here to make sure that there are no other references.

Reply:
Got it. Up to you, as long as we know about this potentially causing problems down the line